playbooks/roles/ocp-customization/README.md

```diff
@@ -20,6 +24,12 @@ Role Variables
 |-------------------------|----------|----------------|---------------------------------------------|
 | workdir                 | no       | ~/ocp4-workdir | Place for config generation and auth files  |
 | rhcos_kernel_options    | no       | []             | List of kernel options for RHCOS nodes eg: ["slub_max_order=0","loglevel=7"] |
+| sysctl_tuned_options    | no       | false       | Set to true to apply sysctl options via tuned operator |
+
+
+`tuned operator options`
+| sysctl_options          | yes       | []             | List of sysctl options to apply. Look at [vars/tuned.yaml](./vars/tuned.yaml) for example |
```

Comment:
Required "yes" only if sysctl_tuned_options is set to true. Same for the next var.